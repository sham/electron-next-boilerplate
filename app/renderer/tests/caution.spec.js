/* eslint-env jest */

import React from 'react';
import Renderer from 'react-test-renderer';
import 'jest-styled-components';

import Caution from '../pages/caution';

test('page Caution', () => {
  const component = Renderer.create(
    <Caution />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});