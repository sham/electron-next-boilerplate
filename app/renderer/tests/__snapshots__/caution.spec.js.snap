// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page Caution 1`] = `
Array [
  <h1>
    Caution
  </h1>,
  <p>
    If you add or remove some pages, 
    <br />
    you need to edit exportPathMap() in next.config.js before exporting.
  </p>,
  <a
    href="/start"
    onClick={[Function]}
  >
    Back
  </a>,
]
`;
